.github/workflows/sycl_precommit_linux.yml

```diff
@@ -71,7 +71,6 @@ jobs:
     if: |
       always()
       && (success() || contains(github.event.pull_request.labels.*.name, 'ignore-lint'))
-      && (github.event.pull_request.head.repo.full_name == 'intel/llvm')
```

Comment:
The version on the main branch (`sycl`) is the following and still contains that check:
```yaml
always()
      && (success() || contains(github.event.pull_request.labels.*.name, 'ignore-lint'))
      && (github.event.pull_request.head.repo.full_name == 'intel/llvm'
          || !contains(needs.detect_changes.outputs.filters, 'ci'))
```

I'm not sure we should remove the check for the repository, IIUC this is a security measure. Probably best to check with the CI maintainers for the main branch first.

Reply:
We removed the `!contains(needs.detect_changes.outputs.filters, 'ci')` check because we want to run tests for merge PR, even if it contains ci changes.
`(github.event.pull_request.head.repo.full_name == 'intel/llvm' || !contains(needs.detect_changes.outputs.filters, 'ci')` means run test if the PR doesn't contain ci changes or the the PR contains ci changes but from `intel/llvm` repo.
As our PRs are from our forks, `github.event.pull_request.head.repo.full_name` is not 'intel/llvm'.
If we keep this portion of the check, none of our PRs get to run any tests.